startup/gui/menus.py

```diff
@@ -135,6 +136,14 @@ def addHelpMenuItems( items ) :
 
 		GafferArnoldUI.CacheMenu.appendDefinitions( scriptWindowMenu, "/Tools/Arnold" )
 
+		scriptWindowMenu.append(
+			"/Tools/Arnold/Populate GPU Cache",
+			{
+				"command" : GafferArnoldUI.GPUCache.populateGPUCache,
+				"active" : lambda : arnold.AiGPUCachePopulateStatus()[1] != 1.0
```

Comment:
> the menu item is only disabled after manually populating the cache, and it's per Gaffer session.

Ah, I had wrongly assumed there was an Arnold API function along the lines of `AiGPUCacheIsPopulated()`. Let's make the menu item always-on as you suggest.

Reply:
Fixed and squashed into 99251579cd85b3e546030ed364c4ff8b1e70db0f